packages/cardpay-sdk/sdk/reward-pool/base.ts

```diff
@@ -63,11 +63,7 @@ export default class RewardPool {
 
   constructor(private layer2Web3: Web3) {}
 
-  async getCurrentPaymentCycle(): Promise<number> {
-    return 2;
-  }
-
-  async getBalance(address: string, rewardProgramId?: string, tokenAddress?: string): Promise<BN> {
```

Comment:
make tokenAddress a compulsory field, so that summing up logic will make sense. If there are two reward tokens, the summing up logic would be wrong